src/cryptoadvance/specterext/spectrum/server_endpoints/templates/spectrum/components/spectrum_info.jinja

```diff
@@ -0,0 +1,85 @@
+        <tr> <td style="text-align: left;">{{ _("Last block hash") }}:</td> <td style="text-align: right"><code style="word-break: break-word;">{{specter.info['bestblockhash']}}</code></td> </tr>
+        <tr> <td style="text-align: left;">{{ _("Mempool Size") }}:</td> <td style="text-align: right;">{{specter.info['mempool_info'].size}} transactions</td> </tr>
+        <tr> <td style="text-align: left;">{{ _("Node uptime") }}:</td> <td style="text-align: right;">~ {{(specter.info['uptime'] / 60 // 60) | int }} {{ _("Hours") }}</td> </tr>
+        {% if specter.info['pruned'] %}
+            <tr> <td style="text-align: left;">{{ _("Automatic pruning") }}:</td> <td style="text-align: right;">{{specter.info['automatic_pruning']}}</td> </tr>
+            <tr> <td style="text-align: left;">{{ _("Prune height") }}:</td> <td style="text-align: right;">{{specter.info['pruneheight']}}</td> </tr>
+            <tr> <td style="text-align: left;">{{ _("Prune target size") }}:</td> <td style="text-align: right;">{{specter.info['prune_target_size']}}</td> </tr>
+        {% endif %}
+    </table> -->
+
+    <p id="total_supply" style="line-height: 2.5;"></p>
+    <div class="row">
+        <button type="button" onclick="fetchTotalSupply()" class="btn centered">
+            {{ _("Run the numbers!") }}
+            <tool-tip width="200px">
+                <h4 slot="title">{{ _("Calculate the total Bitcoin supply") }}</h4>
+                <span slot="paragraph">
+                    {{ _("This will run Bitcoin Core's") }} <code>gettxoutsetinfo</code> {{_("command which will calculate the total amount of Bitcoin's UTXO set.") }}<br>
+                    {{ _("This might take a few minutes...") }}
+                </span>
+            </tool-tip>
+        </button>
+    </div><br>
+    <script>
+        let totalUserBalance = parseFloat(parseFloat("{{ specter.wallet_manager.wallets.values() | sum(attribute='fullbalance') }}").toFixed(8));
+        async function fetchTotalSupply() {
+            document.getElementById('total_supply').innerHTML = `{{ _("Running the numbers... (this might take a few minutes)") }}`;
+            try {
+                const response = await fetch(
+                    "{{ url_for('wallets_endpoint_api.txout_set_info') }}",
+                    {
+                        method: 'GET'
+                    }
+                ).catch((err) => {
+                    showError(err)
+                    return
+                });
+                let result = await response.json();
+                console.log(result)
+                if (result.error) {
+                    showError(result.error)
+                    return
+                }
+                if (totalUserBalance==0) {
+                    document.getElementById('total_supply').innerHTML = `Your wallet holds 0 BTC and so, you're effectively a precoiner! Get off zero! `
```

Comment:
How can we best control copy? Should we extract all strings into a file? I think there will be many places to audit

Reply:
It looks like you're already using the _("i18n") framework, so let's just do that here as well